packages/yum/centos-6/Dockerfile

```diff
@@ -5,28 +5,40 @@ ARG DEBUG
 
 RUN \
   quiet=$([ "${DEBUG}" = "yes" ] || echo "--quiet") && \
-  yum install -y https://packages.groonga.org/centos/groonga-release-latest.noarch.rpm && \
   yum update -y ${quiet} && \
   yum install -y ${quiet} epel-release centos-release-scl && \
+  yum install -y https://packages.groonga.org/centos/groonga-release-latest.noarch.rpm && \
+  yum install -y http://iij.dl.osdn.jp/storage/g/c/cu/cutter/centos/cutter-release-1.3.0-1.noarch.rpm && \
```

Comment:
Why is https://osdn.net/projects/cutter/storage/centos/cutter-release-1.3.0-1.noarch.rpm not used?

Reply:
It should be , but It seems that sometimes fails to fetch from osdn.net.